2025/selection-process.md

```diff
@@ -0,0 +1,183 @@
+proposal a single champion must be nominated for the remainder of the
+process.
+
+Champions and seconds (i.e. organizations that would champion a focus
+area but were not the final selected champion) are recorded for each
+proposal.
+
+There is no assigned limit to the number of proposals one participant
+may champion.
+
+_Alternative_: Each participant is expected to champion a minimum of
+three proposals.
+
+In case different focus areas are put forward with overlapping sets of
+proposals, the participants are encouraged to work out a way to divide
+the proposals into focus areas in a way that satisfies everyone. If
+this is impossible then carrying forward multiple competing focus area
+proposals is permitted, but final selections will only allow each
+proposal to appear in a single focus area.
+
+Any proposal without a champion will not be carried forward.
+
+### Focus Area Proposal Refinement
+
+**Timeline**: October 24th and 31st
+
+Champions gather evidence of the impact of their proposals, using the
+rubric developed earlier. Specific participants may also be asked
+to provide (publicly available) information for all proposals where
+that participant has ownership of the information (e.g. standards
+positions, use counter data).
+
+### Focus Area Proposal Presentation
+
+**Timeline**: November 7th and 14th
+
+Each participant gets 15 minutes of meeting time to present the
+proposals they are championing, and make the case that they should be
+considered a priority for inclusion in Interop 2025.
+
+_Alternative_: Have a longer meeting on either November 7th or November
+14th so that all the proposals are presented in the same week.
+
+### Focus Area Proposal Ranking
+
+**Timeline**: Before December 5th
+
+Participants rank each proposal as P1 (highest priority) to P3 (lowest
+priority), or veto.
+
+There is no preset limit on how many proposals each participant may
+assign each rank.
+
+Initial rankings are submitted in time for the December 5th meeting.
+
+### Focus Area Ranking and Selection
+
+**Timeline**: December 5th, 12th, 19th
+
+Focus areas with any vetoes are eliminated.
+
+Proposals that have strong positive consensus (majority P1 rankings,
+no P3 rankings) are immediately adopted.  Proposals with a strong
+negative consensus (majority P3 rankings, no P1 ranking, or P1 only
+from the champion) are immediately dropped.
+
+Proposals with mixed rankings are ordered according to the number of
+P1 rankings, followed by the number of P2 rankings, followed by the
+number of P3 rankings. Starting from the top of the ordered list
+participants who ranked the proposal lower are given an opportunity to
+explain their concerns, and suggest any adjustments that would change
+their perspective. Proposal champions get final say on whether to
+adjust their proposals in response to this feedback.
+
+Following each meeting participants may adjust their rankings for the
+remaining proposals up or down in response to discussions, any agreed
```

Comment:
As discussed in today's meeting, there's some ambiguity here about whether adjusting up and down is the _only_ way proposals end up being accepted or rejected, in particular as we approach the final decision on Dec 19th.

I think allowing for adjusted rankings in response to argument is good, and that will hopefully move a few things into the accepted state.

I would suggest that all remaining limbo proposals are individually decided by consensus, meaning two supporting and none opposed. This ensures that we have a decision on each proposal that reached this stage, and don't implicitly leave out the bottom N if we run out of time, for example.

Reply:
I've clarified at the end that we have to actually make a decision about any remaining proposals, that is they can't just be ignored. But I'm really hoping that we don't have to use that. If it's clear that everyone is "meh" about a proposal to the extent that we don't even discuss it that seems like a pretty clear sign that it's not important enough to be part of Interop.